vimmo/utils/arg_validator.py

```diff
@@ -1,74 +1,97 @@
 import re
 
-def panel_space_validator(panel_id_value, rcode_value, hgnc_id_value):
-    # Define patterns for Panel_ID, Rcode, and HGNC_ID
-    rcode_pattern = r"^R\d+$"      # Pattern for Rcode: Starts with 'R', followed by digits only
-    panel_pattern = r"^\d+$"       # Pattern for Panel_ID: Matches only digits
-    hgnc_pattern = r"^HGNC:\d+$"   # Pattern for HGNC_ID: Starts with 'HGNC:', followed by digits
 
-    # Validate the format of Panel_ID - must be a number
+def panel_space_validator(panel_id_value, rcode_value, hgnc_id_value):
+    """
+    Validates Panel_ID, Rcode, and HGNC_ID specifically for the panel space.
+    - `Rcode`: Starts with lowercase 'r' followed by digits.
+    - `HGNC_ID`: Starts with 'HGNC:' followed by digits.
+    - `Panel_ID`: Must be numeric.
+    """
+    # Pattern for Rcode: Matches strings starting with 'r' followed by digits (e.g., r123)
+    rcode_pattern = r"^r\d+$"
```

Comment:
Should also accept upper case r and this pattern does not account for the upper case